### DISCUSSION TITLE
Access related items by index

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#4643] [Q&A] [answered]
I know I can do this:
```
{{ related_items }}
  <p>{{ title }}</p>
{{ /related_items }}
```
To walk through the related items.

But now I have 4 related items with whom I want to do different things in the template. So I like to access them directly by index like so (ex.):
```
<div class="main">{{ related_items:0.title }}</div>
<div class="right">{{ related_items:2.title }}</div>
``` 
This does not work. Is there a way to get it to work?

**Comment by duncanmcclean** [comment, marked as answer]
Have you tried `{{ related_items:0:title }}`?

(Note, you'll need the 'antlers' both sides and it's a `:` between 0 and `title`)

**Reply by Sennik** [reply]
I thought I did, but not good enough I think...

It works. Also taxonomies work, like by example categories in a related item: related_items:0:categories:0:title (It is posible to add multiple categories to a related item in my config)

The lack of antlers in the example was a mistake, I had them in the code of course, I have added them in the example above too now ;-)

Thanks for your answer!

**Reply by duncanmcclean** [reply]
No worries! Glad I could help 🙂
